test/mocha/50-records-api.js

```diff
@@ -37,12 +37,18 @@ describe('Records API', () => {
       let actor;
       let ledgerNode;
       let ledgerStorage;
+      let originalGetLatestSummary;
       before(async () => {
         const {id} = mockData.accounts.regularUser.account;
         actor = await brAccount.getCapabilities({id});
         ledgerNode = await brLedgerNode.add(
           actor, {ledgerConfiguration: signedConfig});
         ledgerStorage = ledgerNode.storage;
+        originalGetLatestSummary = ledgerStorage.blocks.getLatestSummary;
+        ledgerStorage.blocks.getLatestSummary = helpers.getLatestBlockSummary;
```

Comment:
```suggestion
        testState = {
          blockHeight: 0,
          updateBlockHeight({blockHeight} ({
            testState.blockHeight = blockHeight;
          }
        };
        ledgerStorage.blocks.getLatestSummary = async () => {
          const {blockHeight} = testState;
          return {eventBlock: {block: {blockHeight}}};
        };
```

And, really, maybe just remove the `updateBlockHeight` function and set `testState.blockHeight = x` below everywhere.